_posts/2018-10-01-why-developers-do-not-like-change.md

```diff
@@ -0,0 +1,75 @@
+---
+author: Sam Davies
+layout: post
+asset-type: post
+title: "Why developers do not like organisational change"
+date: 2018-10-01 00:00:00
+description: A glimpse into the mind of developers when organisational changes are proposed.
+image: 
+    src: /assets/custom/img/blog/2018-10-01-architecture.jpg
```

Comment:
My old office is pictured in this photo 😀

Reply:
Haha it is as if I knew 😉